README.md

```diff
@@ -51,11 +51,18 @@ Each release of Dapr CLI includes various OSes and architectures. These binary v
    * For Linux/MacOS - `/usr/local/bin`
    * For Windows, create a directory and add this to your System PATH. For example create a directory called `c:\dapr` and add this directory to your path, by editing your system environment variable.
 
-### Install Dapr on your local machine (standalone)
+### Install Dapr on your local machine (self-hosted)
+
+In self-hosted mode, dapr can be installed with docker(recommended) or without a docker dependency.
+
+#### With Docker(recommended)
```

Comment:
Phrasing needs to change here. it's "With Redis Container" much more than "With Docker".